protokt-codegen/src/main/kotlin/protokt/v1/codegen/generate/ServiceGenerator.kt

```diff
@@ -431,15 +430,14 @@ private class ServiceGenerator(
         }
 
     private fun <T> pivotPlugin(jvm: T, js: T) =
```

Comment:
Now that we have the concept of a generation target I want to clean this up, but it's complex logic and doesn't have to be cleaned up yet.